src/components/fpl/GameweekStatus.tsx

```
@@ -0,0 +1,331 @@
+            </div>
+          </div>
+        </div>
+      )}
+
+      {/* Clone Information */}
+      <div>
+        <h4 className="font-medium text-gray-900 dark:text-white mb-3 flex items-center">
+          <MdGroups className="text-blue-500 text-xl mr-2" />
+          Vaši klonovi u Top 1 milion
+          <button className="ml-2 text-xs text-blue-600 dark:text-blue-400 hover:underline">
+            Sakrij info o klonovima
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)